### PR TITLE
Add ECR pull task

### PR DESCRIPTION
**Pull request by hunterwerlla**
## Description

Add task to pul images from ECR. This was a weird functionality hole since we are able to push images to repos.

## Related Issue(s), If Filed

Issue #169 

## Checklist

-   [ ] I have read the **README** document
-   [ ] I have read the **CONTRIBUTING** document
-   [ ] My code follows the code style of this project
-   [ ] I have added tests to cover my changes
-   [ ] A short description of the change has been added to the changelog using the script `npm run newChange`

## License

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Tasks/ECRPullImage/TaskParameters.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    imageSource: string
+    imageTag: string
+    imageDigest: string
+    repository: string
+}
+
+export function buildTaskParameters(): TaskParameters {
+    const parameters: TaskParameters = {
+        awsConnectionParameters: buildConnectionParameters(),
+        imageSource: tl.getInput('imageSource', true),
```
Comment (normj) [inline Tasks/ECRPullImage/TaskParameters.ts]: Should these getInput calls be replaced with getInputRequired and getInputOrEmpty?

File: Tasks/ECRPullImage/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "ecrpullimage",
+    "version": "1.1.8",
+    "description": "Pulls a Docker image from Elastic Container Registry",
+    "main": "ECRPullImage.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (normj) [inline Tasks/ECRPullImage/package.json]: What does this do?

File: Tasks/ECRPullImage/TaskParameters.ts (hunk 1 above)
Comment (hunterwerlla) [reply Tasks/ECRPullImage/TaskParameters.ts]: Yep, I made this before I added those functions, they need to be added + the module needs to be gone over when strict is turned on. Will add.

File: Tasks/ECRPullImage/package.json (hunk 2 above)
Comment (hunterwerlla) [reply Tasks/ECRPullImage/package.json]: Good question, this is one of the things I haven't gotten around to deleting (there is one per task), but I think the answer is nothing. I wasn't entirely sure about that when I started working on VSTS, but I think they can all be removed.

Comment (hunterwerlla) [reply Tasks/ECRPullImage/package.json]: Mystery solved, it is used by webpack to package properly, so it is needed.

**Comment by antmanmmt** [comment]
@hunterwerlla Do you have any idea when this will be fully available in Azure DevOps? Thanks

**Pull request merged**